src/core/core.scale.js

```diff
@@ -438,6 +438,9 @@ module.exports = Element.extend({
 			var lineSpace = tickFont.size * 0.5;
 			var tickPadding = me.options.ticks.padding;
 
+			// Store max number of lines used in labels for _autoSkip
+			me.maxLabelLines = tallestLabelHeightInLines;
```

Comment:
We may want to make this `_maxLabelLines` to indicate that it's a private field we don't want to expose in the public API

Reply:
Agree